### PR TITLE
Update Synthetic Dashboards Screenshots + Further Reading Section

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Consistency updates on the screenshots for the Synthetic dashboards and content in the dashboard doc pages. Cleans up legacy page links for the `CI Results Explorer`.

### Motivation

Noticed some new in-app dashboard updates.

### Merge instructions

 
- [x] Please merge after reviewing

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.
- [ ] Check that `cache_enabled` is set to `true` in the `pull_config_preview.yaml` file

### CODE REVIEW

File: content/en/synthetics/dashboards/api_test.md [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
-Click the pink Watchdog icon to open the [**Watchdog Insights**][2] side panel and analyze outstanding anomalies in your application performance or triggered monitor alerts.
+  {{< img src="synthetics/dashboards/api_test_performance_dashboard.png" alt="Out-of-the-box Synthetics API test performance dashboard" style="width:100%" >}}
 
-{{< img src="synthetics/dashboards/api_test_performance_dashboard.png" alt="Out-of-the-box Synthetics API test performance dashboard" style="width:100%" >}}
+- **Events**: View events triggered for all of your API tests and filter for specific tests using the template variables at the top of the dashboard.
+  
+  {{< img src="synthetics/dashboards/api_test_performance_events.png" alt="Events section of the Synthetics API test performance dashboard" style="width:100%" >}}
 
-{{< img src="synthetics/dashboards/api_test_performance_events.png" alt="Events section of the Synthetics API test performance dashboard" style="width:100%" >}}
+Click the pink Watchdog icon to open the [**Watchdog Insights**][2] side panel and analyze outstanding anomalies in your application performance or triggered monitor alerts.
```
Comment (estherk15) [inline content/en/synthetics/dashboards/api_test.md:22]: I couldn't see this when I looked at the app, is there something you need to do to enable this?

File: content/en/synthetics/multistep.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Multistep API tests can run from Datadog [managed](#select-locations) and [priva
 ### Name and tag your test
 
 1. Name your Multistep API test.
-2. Add `env` and other tags to your Multistep API test. You can use these tags to quickly filter through your Synthetic tests on the [Synthetic Monitoring homepage][6].
+2. Add `env` and other tags to your Multistep API test. You can use these tags to quickly filter through your Synthetic tests on the [Synthetic Monitoring & Continuous Testing page][6].
```
Comment (estherk15) [inline content/en/synthetics/multistep.md:47]:
```suggestion
2. Add `env` and other tags to your Multistep API test. You can use these tags to filter through your Synthetic tests on the [Synthetic Monitoring & Continuous Testing page][6].
```

**Review by estherk15 — APPROVED** [review]
Optional suggestions. Don't know if all the sections are the exact same, but might be worth [reuse shortcode](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#General-content-reuse-shortcodes).

File: content/en/synthetics/dashboards/api_test.md (hunk 1 above)
Comment (alai97) [reply content/en/synthetics/dashboards/api_test.md:22]:
```suggestion
```

File: content/en/synthetics/dashboards/browser_test.md [hunk 3]
```diff
@@ -3,29 +3,31 @@ title: Synthetic Browser Test Performance Dashboard
...
+
 - **Synthetic test web performance**: If you have Datadog RUM enabled, use the [RUM integration][2] to examine core web vitals and a list of third-party provider test resources.  
+  
+  {{< img src="synthetics/dashboards/browser_test_web_performance.png" alt="Synthetic test web performance section of the Synthetics Browser test performance dashboard" style="width:100%" >}}
+
 - **Events**: Explore outstanding events from your Synthetic test alerts.
 
+  {{< img src="synthetics/dashboards/browser_test_events.png" alt="Events section of the Synthetics Browser test performance dashboard" style="width:100%" >}}
+
 Click the pink Watchdog icon to open the [**Watchdog Insights**][3] side panel and analyze outstanding anomalies in your application performance or triggered monitor alerts.
```
Comment (alai97) [inline content/en/synthetics/dashboards/browser_test.md:27]:
```suggestion
```

File: content/en/synthetics/dashboards/test_summary.md [hunk 4]
```diff
@@ -2,24 +2,28 @@
...
+
 - **Private locations**: View the number of Synthetic workers by private location, the average concurrency, and average number of pulled tests.
 
-Click the pink Watchdog icon to open the [**Watchdog Insights**][2] side panel and analyze outstanding anomalies in your application performance or triggered monitor alerts.
+  {{< img src="synthetics/dashboards/private_locations.png" alt="Private locations section of the Synthetics test summary dashboard" style="width:100%" >}}
 
-{{< img src="synthetics/dashboards/test_summary_dashboard.png" alt="Out-of-the-box Synthetics test summary dashboard" style="width:100%" >}}
-{{< img src="synthetics/dashboards/test_automation.png" alt="Continuous Testing and CI/CD Integrations section of the Synthetics test summary dashboard" style="width:100%" >}}
-{{< img src="synthetics/dashboards/private_locations.png" alt="Private locations section of the Synthetics test summary dashboard" style="width:100%" >}}
+Click the pink Watchdog icon to open the [**Watchdog Insights**][2] side panel and analyze outstanding anomalies in your application performance or triggered monitor alerts.
```
Comment (alai97) [inline content/en/synthetics/dashboards/test_summary.md:26]:
```suggestion
```

**Pull request merged**